src/main/java/mate/project/exception/CustomGlobalExceptionHandler.java

```diff
@@ -48,6 +48,18 @@ public ResponseEntity<Object> handleEntityNotFoundException(
         return new ResponseEntity<>(body, HttpStatus.NOT_FOUND);
     }
 
+    @ExceptionHandler(RegistrationException.class)
+    public ResponseEntity<Object> handleRegistrationException(
+            RegistrationException ex,
+            WebRequest request) {
+        Map<String, Object> body = new LinkedHashMap<>();
+        body.put("timestamp", ZonedDateTime.now());
+        body.put("message", ex.getMessage());
```

Comment:
The error message is directly from the exception. It would be better to create a user-friendly message.